### PR TITLE
chore：Update README.md,add React-VChart link 

### PR DESCRIPTION
**Pull request by xuanhun**
add React-VChart link to readme

[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [X] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +144,9 @@ $ rush react
 
 | Project                         | Description                   |
 | ------------------------------- | ----------------------------- |
-| [AI-generated Components](todo) | AI-generated chart component. |
+| [React-VChart]([todo](https://github.com/VisActor/VChart/tree/develop/packages/react-vchart)) | React interface for @VisActor/VChart |
```
Comment (kkxxkk2019) [inline README.md:147]: you can remove

Comment (xuanhun) [reply README.md]: It has been corrected,and cn readme also been added @kkxxkk2019

File: README.zh-CN.md [hunk 2]
```diff
@@ -144,7 +144,8 @@ $ rush react
 
 | 项目                 | 介绍                       |
 | -------------------- | -------------------------- |
-| [智能生成组件](todo) | 基于 AI 的智能图表生成组件 |
+| [智能生成组件](https://github.com/VisActor/VChart/tree/develop/packages/react-vchart) | React for  @VisActor/VChart |
```
Comment (kkxxkk2019) [inline README.zh-CN.md:147]: not 智能生成组件， React-VChart？

Comment (xuanhun) [reply README.zh-CN.md:147]: 。。。，Brain broken！！！

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**